frontend/src/utils/index.js

```diff
@@ -42,6 +42,7 @@ import last from 'lodash/last'
 import sample from 'lodash/sample'
 import compact from 'lodash/compact'
 import store from '../store'
+import { sortBy } from 'lodash'
```

Comment:
```suggestion
import sortBy from 'lodash/sortBy'
```

Reply:
This breaks webpack tree shaking https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/